src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterTest.java

```diff
@@ -373,8 +385,8 @@ public void testUniqueProperties() throws Exception {
 
     @Test
     @Ignore("RegexpMultilineCheck has a problem is that violation's "
-            + "line information is not on newline,"
-            + " but on original duplicated property, this maybe solve on context strategy?")
+            + "line information is not on newline/patchedline,"
+            + "this maybe solve on context strategy")
```

Comment:
How these and above changes are related to the main problem your are fixing?

Reply:
because Issue #88 have been fixed, so I want to remove the below Ignore: 

```java
@Ignore("until https://github.com/checkstyle/patch-filters/issues/88 "
            + "when testByConfig can process a directory contains more than one file")
```
but another question is that this check need to be solved in context strategy, otherwise it will fail, so I add the new `@Ignore`.

Reply:
Has to be fixed in https://github.com/checkstyle/patch-filters/issues/104